spec/controllers/users_controller_spec.rb

```diff
@@ -128,6 +128,53 @@ def valid_session
     end
   end
 
+  describe '#feedbacks' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.